packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -75,7 +75,7 @@ function FeedNav(): ReactElement {
         <Tab label={FeedNavTab.History} url="/history" />
       </TabContainer>
       {isMobile && sortingEnabled && isSortableFeed && (
-        <div className="fixed right-0 top-12 my-1 flex h-12 w-20 items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
+        <div className="sticky flex h-12 w-20 -translate-y-12 translate-x-[calc(100vw-100%)] items-center justify-end bg-gradient-to-r from-transparent via-background-default via-40% to-background-default pr-4">
```

Comment:
Don't we get issues now like we had with the bottom bar? (think it was the sticky there right on ios that was the issue)

Reply:
This looks ok from my test on ios as well.

Reply:
My only concern is that design wise now it's different, the bottom line of nav doesn't show anymore where sort icon is located

![IMG_8033](https://github.com/dailydotdev/apps/assets/36197304/9846c1c1-a5d6-430e-89de-7f9e093afbd4)

Reply:
The thing is, sticky is easy to work with when on top of the layout. Shouldn't be an issue in this case and the header has been sticky since then.

Reply:
@nensidosari should be fixable by applying a negative margin or reduce the y-translate.

Reply:
Fixed what I mentioned above on this commit [21fd003](https://github.com/dailydotdev/apps/pull/2955/commits/21fd0030b7a05cd2a55f9553bf1d89b8ea8ce10f)